liveview-android/src/main/java/org/phoenixframework/liveview/domain/factory/ComposableNodeFactory.kt

```diff
@@ -260,4 +280,38 @@ object ComposableNodeFactory {
                 }
             }
             .build()
+
+    private fun buildTopAppBarNode(element: Element, attributes: Attributes): ComposableView =
+        attributes
+            .fold(TopAppBarDTO.Builder()) { builder, attribute ->
+                when (attribute.key) {
+                    "background-color" -> builder.backgroundColor(attribute.value)
+                    "size" -> builder.size(attribute.value)
+                    "height" -> builder.height(attribute.value)
+                    "width" -> builder.width(attribute.value)
+                    "padding" -> builder.padding(attribute.value)
+                    "horizontal-padding" -> builder.horizontalPadding(attribute.value)
+                    "vertical-padding" -> builder.verticalPadding(attribute.value)
+                    else -> builder
+                }
+            }
+            .also { builder ->
+                element.select("text").first()?.let { element ->
+                    builder.textDTO =
+                        buildTextNode(element.attributes(), element.text()) as TextDTO
+                }
```

Comment:
This using `?.` means it might not be found and the `lateinit var textDTO: TextDTO` in `builder` is unsound because `textDTO` might not be initialized if the text is missing.

Reply:
`textDTO` is an optional parameter, I have made it `nullable` and made a null check on its compose call `textDTO?.Compose()`